packages/migrations/src/actions/2025.01.09T00-00-00.legacy-member-scopes.ts

```diff
@@ -0,0 +1,98 @@
+import { z } from 'zod';
+import { type MigrationExecutor } from '../pg-migrator';
+
+const QUERY_RESULT = z.array(z.object({
+  organization_id: z.string(),
+  sorted_scopes: z.array(z.string()),
+  user_ids: z.array(z.string()),
+}));
+
+/**
+ * This migration is going to create a new role for each group of members
+ * that have the same scopes but no role assigned.
+ * 
+ * The role will be named "Auto Role {counter}".
+ * The counter will be reset for each organization.
+ * 
+ * Completes:
+ * https://the-guild.dev/graphql/hive/product-updates/2023-12-05-member-roles
+ * 
+ * Users won't be affected by this change, as they will still have the same scopes.
+ */
+export default {
+  name: '2025.01.09T00-00-00.legacy-member-scopes.ts',
+  noTransaction: true,
+  async run({ sql, connection }) {
+    const queryResult = await connection.query(sql`
+      SELECT
+        organization_id,
+        sorted_scopes,
+        ARRAY_AGG(user_id) AS user_ids
```

Comment:
```suggestion
        organization_id AS "organizationId", 
        sorted_scopes AS "sortedScopes",
        ARRAY_AGG(user_id) AS "userIds"
```

NIT: We could use quotes  and aliases so we don't need to use snake-case in JS 😄

Reply:
NIT